packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -654,6 +655,7 @@ export const ContractType = (() => {
         return {
             cancellation_duration: getArrayDefaultValue(arr_cancellation_range, cancellation_duration),
             cancellation_range_list: arr_cancellation_range.map(d => ({ text: `${getText(d)}`, value: d })),
+            cached_mult_cancellation_list: cached_arr_cancellation.map(d => ({ text: `${getText(d)}`, value: d })),
```

Comment:
nit-pick for refactoring :)
```suggestion
            cached_mult_cancellation_list: cached_multipliers_cancellation.map(mapCancellationRangeList),
```
and we can introduce a variable for the reused callback like this:
```
const mapCancellationRangeList = d => ({ text: `${getText(d)}`, value: d });
```